gateway/clientexec/clientexec.go

```diff
@@ -187,6 +196,17 @@ func (c *clientExec) run(inputPayload []byte, openSessionSpec map[string][]byte)
 		}
 		switch pkt.Type {
 		case pbclient.SessionOpenWaitingApproval:
+			session, err := sessionstorage.FindOne(c, c.sessionID)
+			if err != nil {
+				log.Error("failed obtaining session, err=%v", err)
+			}
+
+			issueInfo := jira.UpdateReviewJiraIssueTemplate{
+				ApiURL:    os.Getenv("API_URL"),
```

Comment:
two things on this:
1. we should use the `appconfig.Get().ApiURL()` method instead of loading it from here to keep the configs coming from one central place
2. maybe this `UpdateReviewJiraIssueTemplate` should not be here but instead inside the `AddReviewCreatedJiraIssue`, because the api url can be retrieved from anywhere and we could send to the `AddReviewCreatedJiraIssue` the session id and it would build this `issueInfo` inside of it. It keeps the design of this integration better componentized, avoiding other parts of our application having to have the responsibility to build something from the jira integration to the jira integration